psh/pshapp/pshapp.c

```diff
@@ -135,6 +135,7 @@ struct psh_redir {
 
 pshapp_common_t pshapp_common;
 
+const char *psh_consolePath;
```

Comment:
Why not place this inside `pshapp_common`?

Reply:
`pshapp_common` is de-facto a static variable and `consolePath` needs to be accessed in `dmesg.c`.

Reply:
No, it isn't.

Reply:
Only uses of this variable are in this file, it is not defined in any header, the same goes for the type `pshapp_common_t`

Reply:
Right, that was my typo - I actually meant `psh_common`.